test/e2e/common/runtime.go

```diff
@@ -172,7 +172,7 @@ while true; do sleep 1; done
 						TerminationMessagePolicy: v1.TerminationMessageFallbackToLogsOnError,
 					},
 					phase:   v1.PodFailed,
-					message: Equal("DONE\n"),
```

Comment:
line 170 calls `echo -n DONE`, so this change seems good for me.
Anyways let's run tests.